data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+	on complete
+		conversation
+			`Your return trip to Aventine is just as exciting, dodging the Archon on the way out of the system.`
+			`	Back in his lab, you report on how the release went. The researcher is pleased to hear that the Void Sprites responded to their presence. "We will give them a few weeks, then try sending another ship to monitor them. Hopefully the Archon will be pacified by this. Meet me in the lab just off the spaceport if you are interested in pursuing this."`
+		"reputation:  Drak " = 1
+
+mission "Remnant: Return the Samples 2"
+	name "Return the Samples 2"
+	description "A researcher on <planet>, a Remnant world, has asked you to return some more samples to Slylandro."
+	source Aventine
+	stopover Slylandro
+	to offer
+		has "license: Remnant"
+		has "Remnant: Return the Samples: done"
+	on offer
+		conversation
+			`Following Plume's directions, you find a lab tucked behind the main area of the spaceport, where a short walkway leads into an underground structure. Stepping inside the door you note a rack of camouflage netting positioned easily at hand. Given what you have learned of their history, it is probably a last ditch preparation in case they needed to hide the entrance from a raid.`
+			`	The lab itself is a microcosm of Remnant development: The structure appears to be some strange semi-organic material, and you can see an eclectic mix of familiar and unfamiliar shapes cluttering the tables. If you had to guess, you would think that this lab was the result of several different families of technology, only one of which is human.`
+			`	Plume and several other Remnant are at the back of the room inspecting a bunch of cocoon-like containers, similar to the ones that carried the last batch of eggs back to Nasqueron. After checking a screen he looks up and seems momentarily startled to see you.`
+			`	"Ah, <first>! You are just in time." Plume chants quickly, his hands unconsciously dancing in a quick series of gestures. "These cocoons are ready to be delivered, this time to Slylandro. Could you drop them off like you did before?"`
+			choice
+				`	"Yes, I would be happy to bring them home."`
+				`	"Sorry, I'm really not interested in braving the Archon a third time."`
+					decline
+
+			`	"Thank you. Like last time, we can't pay you for this. But we are still hopeful that the Archon might let us resume our studies." he follows you out to <ship> and helps you load the cocoons. Just before parting ways he asks "Oh, could you record the void sprites this time? We would like to see the behaviors you mentioned last time.`
+				accept
+	npc
+		government " Drak "
+		system Nenia
+		personality staying heroic nemesis uninterested
+		ship "Archon" "Lifted Lorax"
```

Comment:
Yes, it would. That being said, when I tested it, the Archon was cloaked the entire time, so it seemed like it wasn't even there. And just having an empty system seemed less interesting than having a (former) threat that suddenly wasn't actively a threat anymore.

Reply:
hm, yeah. you could include something like "it seems to be watching", but that's your choice